src/validation/drydep/aero_model_drydep.cpp

```diff
@@ -0,0 +1,197 @@
+        0.43754228462347953e-02};
+
+    EKAT_REQUIRE_MSG(input.has_array("ntot_amode"),
+                     "Required name: ntot_amode");
+    EKAT_REQUIRE_MSG(1 == input.get_array("ntot_amode").size(),
+                     "ntot_amode array should be exactly 1 entry long.");
+    EKAT_REQUIRE_MSG(4 == int(input.get_array("ntot_amode").front()),
+                     "ntot_amode array should be exactly 4.");
+
+    for (std::string s :
+         {"tair", "pmid", "pint", "pdel", "obklen", "ustar", "landfrac",
+          "icefrac", "ocnfrac", "fricvelin", "ram1in", "dt"})
+      EKAT_REQUIRE_MSG(input.has_array(s), "Required name: " + s);
+
+    for (std::string s : {"statq_", "qqcw_"}) {
+      for (int i = 15; i < ConvProc::gas_pcnst; ++i) {
+        const std::string name = s + std::to_string(i + 1);
+        EKAT_REQUIRE_MSG(input.has_array(name), "Required name: " + name);
+        EKAT_REQUIRE_MSG(nlev == input.get_array(name).size(),
+                         name + " array should be exactly 72 entries long.");
+      }
+    }
+
+    for (std::string s : {"dgncur_awet", "wetdens"}) {
+      EKAT_REQUIRE_MSG(input.has_array(s), "Required name: " + s);
+      EKAT_REQUIRE_MSG(nlev * AeroConfig::num_modes() ==
+                           input.get_array(s).size(),
+                       s + " array should be exactly 288 entry long.");
+    }
+
+    for (std::string s : {"tair", "pmid", "pdel"})
+      EKAT_REQUIRE_MSG(nlev == input.get_array(s).size(),
+                       s + " array should be exactly 72 entries long.");
+
+    for (std::string s : {"pint"})
+      EKAT_REQUIRE_MSG(1 + nlev == input.get_array(s).size(),
+                       s + " array should be exactly 72 entries long.");
+
+    for (std::string s : {"obklen", "ustar", "landfrac", "icefrac", "ocnfrac",
+                          "fricvelin", "ram1in", "dt"})
+      EKAT_REQUIRE_MSG(1 == input.get_array(s).size(),
+                       s + " array should be exactly 1 entry long.");
+
+    auto to_dev = [](const std::vector<Real> &vec) {
+      ColumnView dev = mam4::validation::create_column_view(vec.size());
+      auto host = Kokkos::create_mirror_view(dev);
+      for (int i = 0; i < vec.size(); ++i)
+        host[i] = vec[i];
+      Kokkos::deep_copy(dev, host);
+      return dev;
+    };
+
+    haero::ConstColumnView tair = to_dev(input.get_array("tair"));
+    haero::ConstColumnView pmid = to_dev(input.get_array("pmid"));
+    haero::ConstColumnView pint = to_dev(input.get_array("pint"));
+    haero::ConstColumnView pdel = to_dev(input.get_array("pdel"));
+
+    auto state_q_mem =
+        mam4::validation::create_column_view(nlev * ConvProc::gas_pcnst);
+    Diagnostics::ColumnTracerView state_q(state_q_mem.data(), nlev,
+                                          ConvProc::gas_pcnst);
+    {
+      auto host_q = Kokkos::create_mirror_view(state_q);
+      for (int i = 15; i < ConvProc::gas_pcnst; ++i) {
```

Comment:
`15` numeric literal is used a lot as it is the starting index of interstitial aerosols in state_q array. Can we store it in a static constexpr?